src/test/ui/const-eval/union-ice.stderr

```diff
@@ -0,0 +1,33 @@
+error[E0080]: this constant likely exhibits undefined behavior. The rules on what exactly is undefined behavior aren't clear, so this check might be ovezealous. Please open an issue on the rust compiler
+            repository if you believe it should not be considered undefined behavior
```

Comment:
can we move "The rules on what exactly is undefined behavior aren't clear, so this check might be ovezealous. Please open an issue on the rust compiler repository if you believe it should not be considered undefined behavior" to a `note`?